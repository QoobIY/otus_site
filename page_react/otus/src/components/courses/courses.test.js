import React from 'react';
import renderer from 'react-test-renderer';
import Courses from '../courses';

test('test Course component', () => {
    const coursesComponent = renderer.create(
        <Courses courses={[
            {name: 'TestCourse', description: 'Test Description', date: '21.08.1972', },
            {name: 'TestCourse1', description: 'Test Description', date: '22.08.1972', },
            {name: 'TestCourse2', description: 'Test Description', date: '23.08.1972', classes: 'dark'},
        ]}/>
    );
    const coursesJSON = coursesComponent.toJSON();
    expect(coursesJSON).toMatchSnapshot();
});
