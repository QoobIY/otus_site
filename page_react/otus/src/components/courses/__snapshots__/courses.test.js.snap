// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test Course component 1`] = `
<div
  className="courses"
>
  <div
    className="courses__list"
  >
    <div
      className="course undefined"
    >
      <div
        className="course__title"
      >
        TestCourse
      </div>
      <div
        className="course__description"
      >
        Test Description
      </div>
      <div
        className="course__date"
      >
        21.08.1972
      </div>
    </div>
    <div
      className="course undefined"
    >
      <div
        className="course__title"
      >
        TestCourse1
      </div>
      <div
        className="course__description"
      >
        Test Description
      </div>
      <div
        className="course__date"
      >
        22.08.1972
      </div>
    </div>
    <div
      className="course dark"
    >
      <div
        className="course__title"
      >
        TestCourse2
      </div>
      <div
        className="course__description"
      >
        Test Description
      </div>
      <div
        className="course__date"
      >
        23.08.1972
      </div>
    </div>
  </div>
</div>
`;
